### ISSUE TITLE
Feature Request:  ngb-accordion card-header accessibility

### ISSUE THREAD

**Issue by k-spen** [ng-bootstrap/ng-bootstrap#2847]
It seems as though the rendered code looks like this:

```
<div role="tab" id="ngb-panel-13-header" class="card-header ">
<h5 class="mb-0"><button class="btn btn-link collapsed" type="button" aria-expanded="false" aria-controls="ngb-panel-13"> Simple
</button></h5></div>
```

Would it be possible to remove the h5?  This breaks some accessibility rules.  It's not good to skip heading levels and in the context of my page, it goes from h2 to an h5.  Screen readers expect there to be order.  Thanks.

Angular: 6

ng-bootstrap: 3.3.1

Bootstrap: 4

**Comment by pkozlowski-opensource** [comment]
Hmm, interesting. This is actually the markup mandated by Bootstrap itself, see: https://getbootstrap.com/docs/4.1/components/collapse/#accordion-example

As such it should be reported to https://github.com/twbs/bootstrap/issues first. Could you please report the issue in there? We would update our markup as it is considered a bug and fixed on the Bootstrap's end

**Comment by pkozlowski-opensource** [comment]
OK, this got clarified on the Bootstrap side: https://github.com/twbs/bootstrap/issues/27619

tl;dr; we should replace `<h5>` with sth else (probably `<div>`). PRs welcomed!

**Comment by maxokorokov** [comment]
Fixed by #2915, but see #2927 for more accessibility issues
